### PR TITLE
SAK-45306 Add pager to dashboard widgets

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-45306

### CODE REVIEW

**Comment by profmikegreene** [comment]
Should we name the .properties files the same as the webcomponent to help understand the connection between the two? Or does it make more sense to have a single 'assignments' property file that'll have values for anything related to assignments in the webcomponents directory?

**Comment by adrianfish** [comment]
That's not actually a bad idea at all. Conventions are good.

File: webcomponents/tool/src/main/frontend/js/widgets/sakai-dashboard-widget.js [hunk 1]
```diff
@@ -14,40 +15,21 @@ export class SakaiDashboardWidget extends LitElement {
...
-      data: { type: Array },
       editing: { type: Boolean },
     };
   }
 
   constructor() {
 
     super();
-    this.data = [];
+    //this.data = [];
```
Comment (profmikegreene) [inline webcomponents/tool/src/main/frontend/js/widgets/sakai-dashboard-widget.js:25]:
```suggestion
```

**Review by profmikegreene — APPROVED** [review]
I pulled and built it fine, still having big issues with docker and storybook so I can't take a look at it from that tool but I don't think that's an error on the part of this PR. Overall, seems good.

**Review by ern — APPROVED** [review]

**Pull request merged**